packages/web/src/app/clubs/page.tsx

```diff
@@ -1,10 +1,178 @@
 "use client";
 
+import Card from "@sparcs-clubs/web/common/components/Card";
+import PageTitle from "@sparcs-clubs/web/common/components/PageTitle";
+import SectionTitle from "@sparcs-clubs/web/common/components/SectionTitle";
+import Tag from "@sparcs-clubs/web/common/components/Tag";
 import { UseClientProvider } from "@sparcs-clubs/web/common/providers/UseClientProvider";
+import type { FC } from "react";
+import { useCallback, useState } from "react";
+import styled from "styled-components";
+import clubsData from "./data/clubs";
+
+const SizedBox = styled.div<{ width?: number; height?: number }>`
+  width: ${props => (props.width ? `${props.width}px` : `100%`)};
+  height: ${props => (props.height ? `${props.height}px` : `100%`)};
+`;
+
+const SectionTitleContainer = styled.div`
+  display: flex;
+  justify-content: space-between;
+  gap: 20px;
+  flex: 1 0 0;
+`;
+
+const Toggle = styled.div`
+  font-family: ${({ theme }) => theme.fonts.FAMILY.PRETENDARD};
+  font-size: 14px;
+  line-height: 20px;
+  font-weight: ${({ theme }) => theme.fonts.WEIGHT.REGULAR};
+  color: ${({ theme }) => theme.colors.BLACK};
+  text-decoration-line: underline;
+  cursor: pointer;
+`;
+
+const SectionContainer = styled.div`
+  display: grid;
+  grid-template-columns: repeat(4, 1fr);
+  grid-gap: 16px;
+  gap: 16px;
+
+  padding-left: 24px;
+  padding-top: 20px;
+  min-width: 262px;
+  box-sizing: border-box;
+`;
+
+const ClubNameAndMemberCountContainer = styled.div`
+  display: flex;
+  justify-content: space-between;
+  gap: 20px;
+  overflow: hidden;
+`;
+
+const ClubName = styled.div`
+  display: -webkit-box;
+  -webkit-box-orient: vertical;
+  -webkit-line-clamp: 1;
+  word-break: break-all;
+
+  text-overflow: ellipsis;
+  font-family: ${({ theme }) => theme.fonts.FAMILY.PRETENDARD};
+  font-size: 20px;
+  line-height: 24px;
+  font-weight: ${({ theme }) => theme.fonts.WEIGHT.MEDIUM};
+  color: ${({ theme }) => theme.colors.BLACK};
+`;
+
+const ClubMemberCountWithIcon = styled.div`
+  display: flex;
+  align-items: center;
+  gap: 4px;
+
+  font-size: 14px;
+  font-weight: 400;
```

Comment:
```suggestion
  font-weight: ${({ theme }) => theme.fonts.WEIGHT.REGULAR};
```
400이 regular고 500이 medium이고.. 를 저는 가끔 까먹어서 변수로 만들어뒀어요!! 앞으로 font-weight 적용 시 변수 사용해주시면 좋을 것 같습니다!